data-prepper-plugins/elasticsearch/src/test/java/com/amazon/dataprepper/plugins/sink/elasticsearch/ConnectionConfigurationTests.java

```diff
@@ -12,26 +12,30 @@
 import java.util.Objects;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
 public class ConnectionConfigurationTests {
     private final List<String> TEST_HOSTS = Collections.singletonList("http://localhost:9200");
     private final String TEST_USERNAME = "admin";
     private final String TEST_PASSWORD = "admin";
+    private final String TEST_DEFAULT_AWS_REGION = "us-east-1";
     private final Integer TEST_CONNECT_TIMEOUT = 5;
     private final Integer TEST_SOCKET_TIMEOUT = 10;
     private final String TEST_CERT_PATH = Objects.requireNonNull(getClass().getClassLoader().getResource("root-ca.pem")).getFile();
 
     @Test
     public void testReadConnectionConfigurationDefault() {
         final PluginSetting pluginSetting = generatePluginSetting(
-                TEST_HOSTS, null, null, null, null, null, false);
+                TEST_HOSTS, null, null, null, null, false, null, null, false);
```

Comment:
I don't see a test that passes in an AWS region

Reply:
@yadavcbala  has a default which uses "us-east-1", I am changing this in my PR with null and making sure users pass a region.